MapboxNavigation/DefaultStyle.swift

```diff
@@ -13,7 +13,7 @@ extension UIColor {
 extension UIColor {
     // General styling
     fileprivate class var defaultTint: UIColor { get { return #colorLiteral(red: 0.1411764771, green: 0.3960784376, blue: 0.5647059083, alpha: 1) } }
-    fileprivate class var defaultTintStroke: UIColor { get { return #colorLiteral(red: 0.05882352963, green: 0.180392161, blue: 0.2470588237, alpha: 1) } }
+    fileprivate class var defaultTintStroke: UIColor { get { return #colorLiteral(red:0.18, green:0.49, blue:0.78, alpha:1.0) } }
```

Comment:
Do any other properties need to follow suit, like `defaultTint`? In master, the color scheme `#0F2E3F` `#246590` is fairly subtle compared to `#0F2E3F` `#2E7DC7`.